nservicebus/recoverability/index.md

```diff
@@ -39,6 +39,8 @@ By default up to five immediate retries are performed if the message processing
 
 Note: The configured value describes the minimum number of times a message will be retried if its processing consistently fails. Especially in environments with competing consumers on the same queue, there is an increased chance of retrying a failing message more times across the endpoints.
 
+Note: Depending on the concurrency and transactionallity settings of the endpoint immediate retries might happen even if configured to be off. To ensure that no retries is performed the endpoints needs to be configured to be either [non transactional](/nservicebus/transports/transactions.md) or the [concurrency retricted to 1](/nservicebus/operations/tuning.md).
```

Comment:
@andreasohlund - do you think we can document this in a issue with a "Wont Fix" label? I think it will be much better than docs. In the main docs, there won't be any confusion for the readers that won't ever be affected by this, assuming our configuration is just not reliable. Thoughts? cc / @SimonCropp